modules/update-evaluate-availability.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-evaluate-availability_{context}"]
+= Evaluating update availability
+
+The Cluster Version Operator (CVO) periodically queries the OpenShift Update Service for the most recent graph of update possibilities.
+This graph is based on the channel to which the cluster is subscribed.
+If the graph contains paths to OCP versions that are newer than the cluster's current version, the CVO saves these versions into the `available updates` field of its ClusterVersion resource.
```

Comment:
> `available updates`

It's `availableUpdates` in the ClusterVersion.

> the CVO saves these versions into the `available updates` field

If the graph contains paths to OCP versions that are newer than the cluster's current version, the CVO saves these versions into either the `availableUpdates` field or the `conditionalUpdates` field of its ClusterVersion resource.